cmake/configs/modes.cmake

```diff
@@ -137,9 +137,11 @@ set(ubsan_compiler_c_flag "-fsanitize=undefined")
 set(ubsan_compiler_cxx_flag "-fsanitize=undefined")
 
 # MSAN build variant flags.
+# While -fsanitize-memory-track-origins=2 both slows the program, and increases memory needed by msan, it
```

Comment:
The MSan docs say this will slow execution by 1.5-2x but I don't see negative impacts from this. We don't run any MSan tasks in PRs, and only run `compile` in the commit queue which is still fast.

Possibly this reduces coverage of `format-stress-pull-request-test` which is time-limited to 10 minutes but the I think the benefits are well worth it.